gridt/src/app/movements/movements.page.scss

```diff
@@ -1,3 +1,8 @@
 .shortdescription {
-    font-size: 12px;
-}
\ No newline at end of file
+  font-size: 12px;
+}
+
+.custom {
+  width: 10rem;
+  --border-radius: 30px;
+}
```

Comment:
This won't be necessary when implementing the normal button. (Reminder to remove.)